nix/package.nix

```diff
@@ -131,14 +83,16 @@ in
       "script/patches/use-cross-platform-livekit.patch"
     ];
 
+    outputs = ["out"] ++ lib.optional buildRemoteServer "remote_server";
+
     # Dynamically link WebRTC instead of static
     postPatch = ''
       substituteInPlace ../${pname}-${version}-vendor/webrtc-sys-*/build.rs \
         --replace-fail "cargo:rustc-link-lib=static=webrtc" "cargo:rustc-link-lib=dylib=webrtc"
     '';
 
     useFetchCargoVendor = true;
-    cargoHash = "sha256-yzShyOn90U79Ts+sKLhAK6SYiqPX7MKT/9fgF2plgI4=";
+    cargoHash = "sha256-96LOzmiynRycQvIL0+f53UaybuBAbjsf8/vkfR3w7ZM=";
```

Comment:
Can we read `Cargo.lock` directly so we can avoid having to update this hash?

Reply:
We can, I think. I've changed this, but thanks to insane build tooling I still can't test my own changes via builds (lol)